### ISSUE TITLE
Exception when testing Email Notifications System.InvalidOperationException: No recipients have been specified.

### ISSUE THREAD

**Issue by shayfer** [Ombi-app/Ombi#4656]
**Describe the bug**
Update broke gmail notifications.

**To Reproduce**
1. Go to Settings
2. Click on Notifications
3. Click on Email
4. Click Test

**Expected behavior**
Notification test email sent.

**Screenshots**

**Logs (Logs directory where Ombi is located)**

fail: Ombi.Notifications.GenericEmailProvider[0]
      Exception when attempting to send an email
      System.InvalidOperationException: No recipients have been specified.
         at MailKit.Net.Smtp.SmtpClient.SendAsync(FormatOptions options, MimeMessage message, CancellationToken cancellationToken, ITransferProgress progress)
         at MailKit.MailTransport.SendAsync(MimeMessage message, CancellationToken cancellationToken, ITransferProgress progress)
         at Ombi.Notifications.GenericEmailProvider.SendAdHoc(NotificationMessage model, EmailNotificationSettings settings) in /home/runner/work/Ombi/Ombi/src/Ombi.Notifications/GenericEmailProvider.cs:line 112
warn: Ombi.Controllers.V1.External.TesterController[0]
      Exception when testing Email Notifications
      System.InvalidOperationException: No recipients have been specified.
         at MailKit.Net.Smtp.SmtpClient.SendAsync(FormatOptions options, MimeMessage message, CancellationToken cancellationToken, ITransferProgress progress)
         at MailKit.MailTransport.SendAsync(MimeMessage message, CancellationToken cancellationToken, ITransferProgress progress)
         at Ombi.Notifications.GenericEmailProvider.SendAdHoc(NotificationMessage model, EmailNotificationSettings settings) in /home/runner/work/Ombi/Ombi/src/Ombi.Notifications/GenericEmailProvider.cs:line 112
         at Ombi.Controllers.V1.External.TesterController.Email(EmailNotificationSettings settings) in /home/runner/work/Ombi/Ombi/src/Ombi/Controllers/V1/External/TesterController.cs:line 293

**Desktop (please complete the following information):**
 - OS: Ubuntu / Docker Container

**Ombi Version (please complete the following information):**
 - Version 4.16.12
- Media Server Jellyfin
- Database Type: MySQL

**Additional context**
I have many users in User Management with email addresses. Worked great until I updated. 
Any help would be appreciated!

Enable
Enable SMTP Auth
Disable Cert checking. (Toggling doesn't matter)
smtp.gmail.com
587
Password is App password since MFA is turned on.

Used to work.

I used a test website to send a test smtp email with the above settings and it worked just fine.

It seems ombi no longer pulls those email addresses from those accounts to send a test email. I guess I haven't waited until a request is complete to see if it actually sends an email?

**Comment by tidusjar** [comment]
Does your admin user have a email address?

**Comment by shayfer** [comment]
Yes. I am thinking when I upgraded, it was an embarrassingly old version, I removed the email, updated, re-added the email, updated and the test emails works. I thought it might be a weird bug when I updated or something didn't copy right on my end since I noticed that the Welcome email could be sent so the settings were correct and working. I also unchecked the "Do not send Notifications if a User has the Auto Approve permission" toggle. I know that was off before or maybe I had an old version that didn't have it. Not sure. Regardless all good now and Ombi is rock solid and awesome!
